packages/snippet-manager/package.json

```diff
@@ -36,9 +36,9 @@
   },
   "dependencies": {
     "@mongodb-js/devtools-proxy-support": "^0.4.2",
-    "@mongosh/errors": "0.0.0-dev.0",
-    "@mongosh/shell-api": "0.0.0-dev.0",
-    "@mongosh/types": "0.0.0-dev.0",
+    "@mongosh/errors": "2.3.7",
+    "@mongosh/shell-api": "2.3.7",
+    "@mongosh/types": "2.3.7",
```

Comment:
I don't know how this happened, but these don't match with the top-level `"version"` fields

Reply:
whoops, I think I messed up my commit history somehow